libmscore/arpeggio.h

```diff
@@ -79,7 +81,9 @@ class Arpeggio final : public Element {
 
       bool playArpeggio()       { return _playArpeggio; }
       void setPlayArpeggio(bool p) { _playArpeggio = p; }
-
+      
```

Comment:
You removed a blank line and added a line with only whitespace. This should be avoided.